changelog.d/5-internal/WPB-10058

```diff
@@ -0,0 +1,2 @@
+- DB migration for dropping `phone` column from `user` table
+- Tool for deleting phone keys from `user_keys`
```

Comment:
will this run automatically with every upgrade? should this be mentioned here or not?
is important that the first remove-phone release is deployed first if you dont like 5xx errors?

Reply:
I'd argue these belong to features section. Also, making these bullet points will cause the generated markdown to be wrong (or at least a little weird) because each file becomes a bullet point. So just create two files if you need two bullet points.

Reply:
Thanks, guys! Addressed in commit 96c15f09e.

Reply:
done